trtools/dumpSTR/tests/test_dumpSTR.py

```diff
@@ -58,18 +60,51 @@ def args(tmpdir):
     args.verbose = False
     return args
 
+@pytest.fixture
+def testDumpSTRdir(vcfdir):
+    return vcfdir + "/dumpSTR_vcfs"
+
 # Test no such file or directory
-def test_WrongFile(args, vcfdir):
-    fname = os.path.join(vcfdir, "test_non_existent.vcf")
+def test_WrongFile(args, testDumpSTRdir):
+    fname = os.path.join(testDumpSTRdir, "test_non_existent.vcf")
     if os.path.exists(fname):
         os.remove(fname)
     args.vcf = fname
     retcode = main(args)
     assert retcode==1
 
+def test_BadPreexistingFields(args, testDumpSTRdir, capsys):
+    fname = os.path.join(testDumpSTRdir, "bad_preexisting_hrun.vcf")
+    args.vcf = fname
+    retcode = main(args)
+    captured = capsys.readouterr()
+    assert "HRUN" in captured.err
+
+    fname = os.path.join(testDumpSTRdir, "bad_preexisting_het_hwep.vcf")
+    args.vcf = fname
+    retcode = main(args)
+    captured = capsys.readouterr()
+    assert "HWEP" in captured.err and "HET" in captured.err
+
+    fname = os.path.join(testDumpSTRdir, "bad_preexisting_filter_ac_refac.vcf")
+    args.vcf = fname
+    retcode = main(args)
+    captured = capsys.readouterr()
+    assert ("FILTER" in captured.err and "AC" in captured.err
+            and "REFAC" in captured.err)
+
+def test_WorrisomePreexistingFilter(args, testDumpSTRdir, capsys):
+    fname = os.path.join(testDumpSTRdir, "worrisome_preexisting_filter.vcf")
```

Comment:
Can you maybe add a couple of lines of comment for each test to say what's being tested?

Reply:
I added comments to all the new tests.